src/Compilers/CSharp/Portable/Symbols/Source/SourceEventAccessorSymbol.cs

```diff
@@ -54,10 +55,14 @@ internal abstract class SourceEventAccessorSymbol : SourceMemberMethodSymbol
 
             this.MakeFlags(
                 isAdder ? MethodKind.EventAdd : MethodKind.EventRemove,
+                RefKind.None,
                 @event.Modifiers,
                 returnsVoid: false, // until we learn otherwise (in LazyMethodChecks).
+                hasAnyBody: false,
```

Comment:
not sure the best thing to do here.  we weren't setting this before.  but conceptually i don't think this shoudl ever be false if isExpressionBodied is true.  Guessing this is simply never matters for this type.  inclear if we should default to 'false' in that case, or attempt to set a reasonable value.